src/components/radio-button/RadioButton.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { View, Text, Pressable } from 'react-native';
+import { RadioButtonProps } from './radio-button.types';
+import { styles } from './radio-button.styles';
+
+const RadioButton: React.FC<RadioButtonProps> = ({
+  value,
+  selected,
+  onPress,
+  label,
+  disabled = false,
+  kind = 'primary',
+  containerStyle,
+  labelStyle,
+}) => {
+  const getColors = (type: String) => {
```

Comment:
Can you see how other components are keying the styles based on kind?

[Button](https://github.com/jalantechnologies/react-native-template/tree/main/src/components/button) component is a good example